target/linux/airoha/files/drivers/pwm/pwm-airoha.c

```diff
@@ -76,21 +85,18 @@ struct airoha_pwm {
 /* Duty cycle is relative with 255 corresponding to 100% */
 #define DUTY_FULL	255
 
-static u32 airoha_pwm_rmw(struct airoha_pwm *pc, void __iomem *addr,
+static int airoha_pwm_rmw(struct airoha_pwm *pc, int offset,
```

Comment:
@Ansuel maybe we can just get rid of airoha_pwm_rwm() and just use regmap_update_bit() directly